spec/EMBEDDED_PROTOCOL_VERSION

```diff
@@ -1 +1 @@
-2.3.0
+3.0.0-dev
```

Comment:
As we don't have any other planned protocol changes for next version right now, I suggest just change this to a stable version, so that it will be tagged automatically as we merge this PR.

Reply:
Thanks- updated in https://github.com/sass/sass/pull/3728/commits/63c45082729730eb0c74cb83c0536e13fc7d9b1c